url.bs

```diff
@@ -116,17 +116,17 @@ error.
 <h3 id=percent-encoded-bytes>Percent-encoded bytes</h3>
 
 <p>A <dfn>percent-encoded byte</dfn> is U+0025 (%), followed by two <a>ASCII hex digits</a>.
-Sequences of <a lt="percent-encoded byte">percent-encoded bytes</a>, after conversion to bytes,
+Sequences of <a lt="percent-encoded byte">percent-encoded bytes</a>, <a>isomorphic encoded</a>,
 should not cause <a>UTF-8 decode without BOM or fail</a> to return failure.
```

Comment:
This sentence is pretty unclear to me both before and after the change, so I'm unsure how to evaluate whether "isomorphic encoded" was what is meant. The usage of "should" also seems off.

Reply:
Thanks, that was indeed wrong.

Reply:
I'll file a follow-up to investigate further clarification.